app/pages/home-for-user/news-section.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import { Link } from 'react-router';
+import ProjectCard from '../../partials/project-card';
+import Publications from '../about/publications-page';
+
+import style from './news-section.styl';
+void style;
+
+const NewsSection = React.createClass({
+
+  getDefaultProps() {
+    return {
+      newDatasets: [],
+    };
+  },
+
+  componentDidMount() {
+    this.fetchProjects();
+    this.recentPublications();
+  },
+
+  getInitialState() {
+    return {
+      avatars: {},
+      projects: [],
+      publications: [],
+    };
+  },
+
+  recentPublications() {
+    const articles = []
+    for (var category in Publications.articles()) {
+      Publications.articles()[category].map((project) =>{
+        articles.push(...project.publications)
+      });
+    };
+    const newestPublications = articles.sort((a, b) => {
+      return new Date(b.date) - new Date(a.date)
+    }).splice(0, 3);
+    return this.setState({
+      publications: newestPublications,
+    });
+  },
+
+  fetchProjects() {
+    apiClient.type('projects').get({
+      sort: '-launch_date',
+      launch_approved: true,
+      page_size: 1,
+      include: ['avatar'],
+    })
+    .then((projects) => {
+      this.setState({
+        projects,
+      });
+
+      return Promise.all(projects.map((project) => {
+        return project.get('avatar')
+        .catch(() => {
+          return null;
+        })
+        .then((avatar) => {
+          this.state.avatars[project.id] = avatar;
+          this.forceUpdate();
+        });
+      }));
+    })
+  },
+
+  renderPublication(article) {
```

Comment:
The `<div>` this is returning needs a `key` property since it is in the map function in the render.